src/main/drivers/serial_escserial.c

```diff
@@ -1011,7 +1018,7 @@ bool escEnablePassthrough(serialPort_t *escPassthroughPort, const motorDevConfig
                     closeEscSerial(ESCSERIAL1, mode);
                     return true;
                 }
-                if (mode==PROTOCOL_BLHELI) {
+                if (mode==PROTOCOL_BLHELI || mode==PROTOCOL_KISS || mode==PROTOCOL_KISSALL) {
```

Comment:
protocol _KISSALL was renamed to _CASTLE in this PR (src/main/cli/cli.c:3940), yet _KISSALL was added to this if statement.

Reply:
https://github.com/betaflight/betaflight/blob/698b8616f930840b7b85a15549bcd7f67e76f641/src/main/drivers/serial_escserial.h#L32-L39

Reply:
This IMO enables echo for kiss protocols too.

Reply:
Only useable is ki 255, flashing esc one by one is really not interested and should not be used. KISS is being converted to KISSALL by presense of motor output 255. kiss protocol requires echoing data back.